site/ts/grader-submission.ts

```diff
@@ -20,7 +20,26 @@ declare global {
 
 const warning_banner = document.getElementById('submission-mode-warning');
 
+function initialsubmissionmode() {
+    const radioBulk = document.getElementById('radio-bulk') as HTMLInputElement;
+    const pdfSubmitButton = document.getElementById('pdf-submit-button');
+
+    if (radioBulk.checked) {
```

Comment:
Not all gradeables have the bulk upload option right? 

Would checking if this is not null or something like that fix the cypress ?

Reply:
This was the problem. Good catch thanks :)